data-prepper-plugins/opensearch/src/main/resources/index-template/logs-otel-v1-index-standard-template.json

```diff
@@ -6,58 +6,129 @@
       "_source": {
         "enabled": true
       },
+      "dynamic_templates": [
+        {
+          "integer_resource_attributes": {
```

Comment:
shouldn't this be `long_resource_attributes`
I think the question is already raised in other comments. can we address that in every signal type.